### ISSUE TITLE
Ability to setup a VM startup/shutdown order in the cluster

### ISSUE THREAD

**Issue by jay7x** [jay7x/puppet-lima#14]
It'd be nice to have a VM startup/shutdown order in the cluster. That allows to do staged cluster startup/shutdown.

Imagine we have 3 VM with `node1` having order of 10 and other 2 VMs having order of 100. Then `node1` should be started first and the other nodes should be started after the 1st one in parallel. Shutdown sequence should happen in a reversed order.

**Comment by jay7x** [comment]
I guess `priority` is better (0 - default, higher value => start earlier)
